### PR TITLE
Add profile media

### PR DESCRIPTION
**Pull request by JanPietrzyk**

### CODE REVIEW

File: src/Merchants/Content/Merchant/Api/ProfileController.php [hunk 1]
```diff
@@ -26,15 +28,25 @@ class ProfileController
...
 
-    public function __construct(EntityRepositoryInterface $merchantReepository, DataValidator $dataValidator)
-    {
+    /**
+     * @var StorefrontMediaUploader
+     */
+    private $uploader;
+
+    public function __construct(
+        EntityRepositoryInterface $merchantReepository,
```
Comment (PaddyS) [inline src/Merchants/Content/Merchant/Api/ProfileController.php]: `$merchantRepository`

Comment (JanPietrzyk) [reply src/Merchants/Content/Merchant/Api/ProfileController.php]: If it's just that, I will change it in the next PR, ok?

Comment (JanPietrzyk) [reply src/Merchants/Content/Merchant/Api/ProfileController.php]: On the other hand, I just added delete here too :)

File: src/Merchants/Content/Merchant/MerchantEntity.php [hunk 2]
```diff
@@ -135,6 +136,11 @@ class MerchantEntity extends Entity
      */
     protected $orders;
 
+    /**
+     * @var MediaCollection
```
Comment (shyim) [inline src/Merchants/Content/Merchant/MerchantEntity.php]: Can be null

Comment (JanPietrzyk) [reply src/Merchants/Content/Merchant/MerchantEntity.php]: Ah, forgot that collections are not just instantiated emptily....

**Pull request merged**